### ISSUE TITLE
System.TypeLoadException when PublishTrimmed = true and PublishAot = false

### ISSUE THREAD

**Issue by MartinKuschnik** [MartinKuschnik/WmiLight#15]
The Type `System.StubHelpers.ObjectMarshaler` is trimmed. This causes a `System.TypeLoadException`.

**Comment by MartinKuschnik** [comment]
To fix this issue, the `BuiltInComInteropSupport` must be enabled.

**Comment by danielo-unity3d** [comment]
```
$ dotnet publish -p:PublishBuildType=Trimmed
ILLink : Trim analysis error IL2026: Internal.Runtime.InteropServices.ComActivator.GetClassFactoryForTypeInternal(ComActivationContextInternal*)
: Using member 'Internal.Runtime.InteropServices.ComActivator.GetClassFactoryForTypeImpl(ComActivationContextInternal*, Boolean)' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. Built-in COM support is not trim compatible. https://aka.ms/dotnet-illink/com
C:\Users\user\.nuget\packages\microsoft.net.illink.tasks\8.0.3\build\Microsoft.NET.ILLink.targets(87,5): error NETSDK1144: Optimizing assemblies for size failed. Optimization can be disabled by setting the PublishTrimmed property to false.
```

**Comment by MartinKuschnik** [comment]
fixed with 52e56ad12ed959da2eef2e9999ef2b07d3b922af

**Comment by MartinKuschnik** [comment]
Fixed with Release [Version 6.0.0](https://github.com/MartinKuschnik/WmiLight/releases/tag/v6.0.0)
